Sources/Purchasing/Purchases/PurchasesOrchestrator.swift

```diff
@@ -413,9 +413,11 @@ final class PurchasesOrchestrator {
                     productIdentifier: product.id,
                     error: error
                 ))
+                let publicError = ErrorUtils.purchasesError(withUntypedError: error).asPublicError
+                let userCancelled = publicError.code == ErrorCode.purchaseCancelledError.rawValue
```

Comment:
Oh I see why you needed this now. We need to compare the error domain too though, I'll fix that.

Reply:
I'm changing it to this:
```suggestion
                let userCancelled = publicError.isCancelledError
```